src/themes/shared/slotRecipes/energyLabel.ts

```diff
@@ -0,0 +1,102 @@
+import { defineSlotRecipe } from '@chakra-ui/react';
+
+export const energyLabelRecipe = defineSlotRecipe({
+  className: 'chakra-energy-label',
+  slots: ['root', 'triangle', 'textWrapper', 'text'],
+  base: {
+    root: {
+      '--A': '#4CA651',
```

Comment:
for readability myabe something like `--labelColor-A` to match the colors object in place before